### ISSUE TITLE
Download + Retrieving Info from YouTube doesn't work

### ISSUE THREAD

**Issue by NathanBnm** [bernardodsanderson/vido#30]
There is just a error message, I will post a screenshot soon.

**Comment by NathanBnm** [comment]
![image](https://user-images.githubusercontent.com/45366162/52739896-28767400-2fd2-11e9-9765-5cb511eaa9e3.png)
![image](https://user-images.githubusercontent.com/45366162/52739929-375d2680-2fd2-11e9-8cfc-e93ae36b2ad2.png)
It just doesn't fit in the window!

**Comment by ryonakano** [comment]
It seems this is because `youtube-dl`, which is used in this app, is not up-to-date on your system. I confirmed I could get the info of the same URL as you tried to and download it by updating `youtube-dl` as described [here](https://blog.madadipouya.com/2016/02/27/how-to-update-youtube-dl-in-ubuntu/):

![screenshot from 2019-02-14 10-57-55](https://user-images.githubusercontent.com/26003928/52757361-be98b180-3047-11e9-87b4-846a98fe2a1b.png)

**Comment by NathanBnm** [comment]
Yes, it's working now, but is there a way that the app does theses operations automatically? Users haven't to do that by themselves when they want to use the app.

**Comment by ryonakano** [comment]
@NathanBnm Then I think it would be good to remove `youtube-dl` from dependencies in the debian control file (because its version in the apt repository is usually out-of-date) and instead add a script which executes [the command](https://blog.madadipouya.com/2016/02/27/how-to-update-youtube-dl-in-ubuntu/) during the installation of this app. (I think it's not a good way to execute that command within the app, because it requires a root privilege, thus the app have to be executed with `sudo` command every time it runs)

To do this, I suppose it would be good to create `debian/postinst` in this repository with the following content:

```sh
#!/bin/bash -e

sudo wget https://yt-dl.org/latest/youtube-dl -O /usr/local/bin/youtube-dl
sudo chmod a+x /usr/local/bin/youtube-dl
```

and remove `youtube-dl` from these lines:

https://github.com/bernardodsanderson/vido/blob/dfbcb9dd4b2086a3f54e4014b247fbe13a199e59/debian/control#L9

https://github.com/bernardodsanderson/vido/blob/dfbcb9dd4b2086a3f54e4014b247fbe13a199e59/debian/control#L15

I tried making these changes in my local, building a `.deb` file of this app and installing it. And then I confirmed the latest version of `youtube-dl` is installed during the installation of the app and the video you've tried to is able to be downloaded.

Reference: https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html

**Comment by NathanBnm** [comment]
Fixed in #32. Closing.
